crates/modelardb_manager/src/metadata.rs

```diff
@@ -162,7 +162,8 @@ impl MetadataManager {
             .execute(
                 format!(
                     "CREATE TABLE {}_compressed_files (file_name BYTEA PRIMARY KEY, field_column INTEGER,
-                     start_time INTEGER, end_time INTEGER, min_value REAL, max_value REAL)",
+                     folder_path TEXT, size INTEGER, created_at INTEGER, start_time INTEGER,
```

Comment:
Combined them into `file_path`. Also looked into completely removing `CompressedFile` but decided to keep it since it makes the interface for the functions that need all the information in `CompressedFile` cleaner.